builders/xcm/fees.md

```diff
@@ -79,35 +79,43 @@ Although Polkadot doesn't currently use database weight units to calculate costs
 | RocksDB (default) | {{ networks.polkadot.rocks_db.read_weight }}  | {{ networks.polkadot.rocks_db.write_weight }}  |
 |     ParityDB      | {{ networks.polkadot.parity_db.read_weight }} | {{ networks.polkadot.parity_db.write_weight }} |
 
-With the instruction weight cost established, you can calculate the cost of the instruction in DOT. 
+With the instruction weight cost established, you can calculate the cost of each instruction in DOT. 
 
-In Polkadot, the [`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/weights/extrinsic_weights.rs#L56){target=_blank} is set to `{{ networks.polkadot.extrinsic_base_weight.display }}` which is mapped to 1/10th of a cent. Where 1 cent is `10^10 / 10,000`. Therefore, a constant exists in the formula for calculating the final fee in DOT:
+In Polkadot, the [`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/weights/extrinsic_weights.rs#L56){target=_blank} is set to `{{ networks.polkadot.extrinsic_base_weight.display }}` which is [mapped to 1/10th](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs#L87){targer=blank} of a cent. Where 1 cent is `10^10 / 100`. 
+
+Therefore, to calculate the cost of executing an XCM instruction, you can use the following formula:
 
 ```
-Planck-DOT-Weight =  PlanckDOT-Mapped * (1 / ExtrinsicBaseWeight)
+XCM-DOT-COST = XCMInstrWeight * DOTWeightToFeeCoefficient
 ```
 
-Where the constant is calculated as follows:
+Where `DOTWeightToFeeCoefficient` is a constant (map to 1 cent), and can be calculated as:
 
 ```
-Planck-DOT-Weight = (10^10 / 10000) * (1 / {{ networks.polkadot.extrinsic_base_weight.numbers_only }})
+KSMWeightToFeeCoefficient = ( 10^10 / ( 10 * 100 )) * ( 1 / DOTExtrinsicBaseWeight )
 ```
 
-As a result, `Planck-DOT-Weight` is equal to `{{ networks.polkadot.xcm_instruction.planck_dot_weight }} Planck-DOT`. Now, you can begin to calculate the final fee in DOT, using `Planck-DOT-Weight` as a constant, and `TotalWeight` as the variable:
+Using the actual values:
 
 ```
-Total-Planck-DOT = TotalWeight * Planck-DOT-Weight
-DOT = Total-Planck-DOT / DOTDecimalConversion
+DOTWeightToFeeCoefficient = (10^10 / ( 10 * 100 * {{ networks.polkadot.extrinsic_base_weight.numbers_only }})
+```
+
+As a result, `DOTWeightToFeeCoefficient` is equal to `{{ networks.polkadot.xcm_instruction.planck_dot_weight }} Planck-DOT`. Now, you can begin to calculate the final fee in DOT, using `DOTWeightToFeeCoefficient` as a constant and `TotalWeight` as the variable:
+
+```
+XCM-Planck-DOT-COST = TotalWeight * DOTWeightToFeeCoefficient
```

Comment:
shouldn't `Planck` just be capitalized too? Either that or `COST` should be lower case?

Reply:
True, I've set `COST` to `Cost`